.github/workflows/ci_checks.yml

```diff
@@ -160,13 +170,3 @@ jobs:
           USER_EMAIL: ${{ secrets.USER_EMAIL }}
           BRANCH_NAME: ${{ github.event.pull_request.head.ref }}
         run: .github/scripts/amend_and_push.sh
-
-  qa_purge_env:
-    runs-on: [ubuntu-latest]
-    needs: [qa_lint, qa_unit_tests, qa_android_tests, mobius_migration_tests]
```

Comment:
So I am guessing since purge is only needed for Android tests we longer have to depend on all of the other jobs?

Reply:
Yup.